scripts/make_release.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
```

Comment:
note: #!/usr/bin/env bash
is more portable IMHO
ref: https://www.cyberciti.biz/tips/finding-bash-perl-python-portably-using-env.html